packages/export-adapters/draftjs-to-digital-paper-edit/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Convert DraftJS to Digital Paper Edit format 
+ * More details see
+ * https://github.com/bbc/digital-paper-edit
+ */
+export default (blockData) => {
+    let result = {words: [], paragraphs: []};
+
+    blockData.blocks.forEach((block, index)=>{
+        if(block.data.words !== undefined){
```

Comment:
This branch needs a linting first - `npm run lint:fix` should sort it 👍

Reply:
done, thanks